### PR TITLE
Build on Github Actions

### PR DESCRIPTION
**Pull request by giorgiosironi**
For https://github.com/libero/publisher/issues/340

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    - name: Smoke tests
+      run: |
+        set -e
+        TARGET=prod .travis/smoke-test.sh
+        TARGET=dev .travis/smoke-test.sh
+
+    - name: Lint
+      run: make lint
+
+    - name: Test
```
Comment (giorgiosironi) [inline .github/workflows/ci.yml:32]: downstream steps are not executed if a previous one fails, at the moment

**Comment by giorgiosironi** [comment]
Opened on the origin repository as forks seem to be poorly supported at the moment.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (thewilkybarkid) [reply .github/workflows/ci.yml:32]: What's the cost of separate actions? Multiple Docker builds?

Comment (giorgiosironi) [reply .github/workflows/ci.yml:32]: Seems multiple jobs, which start in isolated environments and hence Docker builds; unless the image can be shared as an artifact which is possible in principle but likely convoluted. Jobs enable parallelization as they can run on separate environments (actual VMs?)

We may also runn stuff in parallel at the microlevel (`make`, `xargs`) but they will be seen as a single step so doesn't make the failures clear. In general there are some dependencies between these steps, if the smoke tests do not run also `analyse.sh` likely won't (besides dependencies towards `make build` which previously was separated in Travis CI's `install` section).

**Comment by giorgiosironi** [comment]
Possibly the missing event `push` ~`pull_request`~ is causing the fork problems. Will close this and keep working from the fork.

**Pull request closed without merging**